integration_tests/src/main/python/array_test.py

```diff
@@ -740,3 +740,20 @@ def test_flatten_array(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark: unary_op_df(spark, data_gen).selectExpr('flatten(a)')
     )
+
+# No NULL keys are allowed
+data_gen = [IntegerGen(nullable=False), StringGen(nullable=False),
+            StructGen(nullable=False,children=[('a',IntegerGen())]), DateGen(nullable=False),
+            DoubleGen(nullable=False), TimestampGen(nullable=False)]
+
+@pytest.mark.parametrize('data_gen', data_gen, ids=idfn)
+def test_map_from_arrays(data_gen):
+    # min_length and max_length is fixed because map_from_arrays expects same sized array for keys and values
+    # NULL rows are valid
+    gen = StructGen(
+        [('a', ArrayGen(data_gen, nullable=True, min_length=10, max_length=10)), ('b', ArrayGen(data_gen, nullable=True, min_length=10, max_length=10))], nullable=False)
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: gen_df(spark, gen).selectExpr(
+            'map_from_arrays(a, b)'),
+        conf={'spark.sql.mapKeyDedupPolicy': 'LAST_WIN'}
+    )
```

Comment:
Can we have a test to verify that the DEDUP key checks work like with https://github.com/NVIDIA/spark-rapids/blob/be34c6a97004d46ca2541e214552fecb5a172ddd/integration_tests/src/main/python/map_test.py#L293-L331

Reply:
I agree, done!